ci/test/00_setup_env_win64.sh

```diff
@@ -7,7 +7,7 @@
 export LC_ALL=C.UTF-8
 
 export CONTAINER_NAME=ci_win64
-export DOCKER_NAME_TAG=ubuntu:18.04  # Check that bionic can cross-compile to win64 (bionic is used in the gitian build as well)
+export DOCKER_NAME_TAG=ubuntu:18.04  # Check that bionic can cross-compile to win64
```

Comment:
Any reason to not bump the ci configs as well? They should stay in-sync if possible.

Reply:
For the win config specifically, per the OP, if we bump to Focal, running the [security tests](https://github.com/bitcoin/bitcoin/blob/master/ci/test/00_setup_env_win64.sh#L14) wont work, so that would have to be removed. If you're happy to move them (all?) to Focal we can do that.

Reply:
My understanding is that the two cross builds (windows, macos) are used as proxies for the gitian build. If you keep this one at bionic, it will use gcc8, while the gitian build uses gcc9. Ideally, I think they should be as close as possible to each other.


(Unrelated fun fact: We don't have any linux cross builds in the ci that mimic the gitian linux cross builds)

Reply:
Ok. I've changed the macOS and Windows CIs to now use Focal.